.circleci/config.yml

```diff
@@ -250,7 +250,7 @@ jobs:
             mkdir -p ~/.ssh
             ssh-keyscan github.com >> ~/.ssh/known_hosts
             echo "Comparing against $RELEASE_TAG"
-            # Github has phased out the git protocol so we ensure that we use
```

Comment:
they did phase out git protocol too. 

the line below changes git:// to https: